casper/src/test/scala/coop/rchain/casper/util/GenesisBuilder.scala

```diff
@@ -116,6 +120,7 @@ object GenesisBuilder {
           numberOfActiveValidators = 100,
           validators = bonds.map(Validator.tupled).toSeq,
           posMultiSigPublicKeys = defaultPosMultiSigPublicKeys,
+          posVaultPubKey = defaultPosVaultPubKey,
```

Comment:
Keep `posMultiSigPublicKeys` and `posMultiSigQuorum` parameters together. They are part of the same functionality.